### PR TITLE
[DOCS-7551] Add Alfresco Digital Workspace 4.4 docs

### PR DESCRIPTION
**Pull request by aritrihore**

### CODE REVIEW

**Review by Prosune — DISMISSED** [review]
LGTM

File: digital-workspace/4.3/develop/manage.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+## Clone and launch the front-end application
+
+Once Content Services is up and running, you must make the source code of the project available locally in your development environment. If you're an Alfresco customer or partner, you can get a local copy of the project by opening a request in [Hyland Community](https://community.hyland.com/tskb){:target="_blank"}.
+
+1. In the `alfresco-apps` directory, create a file named `.env` with the following content.
+
+    Enter the Content Services URL as the `APP_CONFIG_ECM_HOST` value.
+
+    ```bash
+    ACA_BRANCH="2.10.0"
```
Comment (MichalKinas) [inline digital-workspace/4.3/develop/manage.md:27]: I believe that's no longer used

File: digital-workspace/4.3/develop/manage.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    APP_CONFIG_ECM_HOST="http://localhost:8080"
+    APP_CONFIG_PROVIDER="ECM"
+    APP_CONFIG_AUTH_TYPE="BASIC"
+    APP_CONFIG_PLUGIN_AOS=false
+    APP_CONFIG_PLUGIN_CONTENT_SERVICE=true
+    APP_CONFIG_PLUGIN_PROCESS_SERVICE=false
+    APP_CONFIG_PLUGIN_AI_SERVICE=false
+    ```
+
+    > **Note:** The Digital Workspace 4.3.x uses the Content Application version 4.0.0.
```
Comment (MichalKinas) [inline digital-workspace/4.3/develop/manage.md:38]: Actually ADW 4.3.x uses ACA 4.3.0 not 4.0.0

File: digital-workspace/4.3/develop/manage.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+    | Distribution | Description |
+    | ------------ | ----------- |
+    | `content-ee` | *Default.* The Digital Workspace with the Process Services extension |
+    | `content-ee-apa` | The Digital Workspace with the Process Automation extension |
+
+2. Select the command to start the application:
+
+    ```bash
+    npm start <content-ee|content-ee-apa> [prod]
```
Comment (MichalKinas) [inline digital-workspace/4.3/develop/manage.md:112]: `npm start content-ee-apa` won't work with OOTB ADW, at least we don't have it right now in our repo

File: digital-workspace/4.3/develop/manage.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+    ```
+
+    > **Note:** For the Alfresco Content Application or any other ADF-based application, the command is `npm start`.
+
+3. Select the command to build the application:
+
+    ```bash
+    npm run build <content-ee|content-apa> [prod]
```
Comment (MichalKinas) [inline digital-workspace/4.3/develop/manage.md:120]: Same here for `content-ee-apa`

File: digital-workspace/4.3/develop/deploy.md [hunk 5]
```diff
@@ -0,0 +1,264 @@
...
+Deploy Digital Workspace using the following steps:
+
+1. Request a local copy of the `alfresco-apps` GitHub repository project from [Hyland Community](https://community.hyland.com/tskb){:target="_blank"}.
+
+    There are two distributions available to run with the Digital Workspace:
+
+    | Distribution | Description |
+    | ------------ | ----------- |
+    | `content-ee` | *Default.* The Digital Workspace with the Process Services extension |
+    | `content-ee-apa` | The Digital Workspace with the Process Automation extension |
```
Comment (MichalKinas) [inline digital-workspace/4.3/develop/deploy.md:77]: Same here for APA

File: digital-workspace/4.3/develop/deploy.md [hunk 6]
```diff
@@ -0,0 +1,264 @@
...
+    http://localhost:8080/alfresco
+    ```
+
+* `Node.js` 18.x
+
+> **Note:** All Angular development is done using the Typescript language.
+
+### Generate the app
+
+1. To ensure you have Yeoman installed, open a command prompt and enter:
```
Comment (MichalKinas) [inline digital-workspace/4.3/develop/deploy.md:174]: @DenysVuika do we still support Yeoman?

File: digital-workspace/latest/config/index.md [hunk 7]
```diff
@@ -61,6 +61,16 @@ You can edit the columns you want to see in any of the file views in the Digital
 
     The columns are no longer visible in the view.
 
+### Drag and drop
```
[the remark this answers is not in the document]
Comment (Prosune) [reply digital-workspace/latest/config/index.md:64]: Mentioned this in a note. Please review and provide your feedback.

File: digital-workspace/latest/config/index.md [hunk 8]
```diff
@@ -69,7 +69,9 @@ You can manually edit the position of the columns in any of the file views in Di
 
 2. In the view you want to adjust, drag and drop the columns to your desired position. 
 
-    > **Note:** The position does not revert back to the original view, once you click away.
+    > **Note:**
+    > * A column can be dragged to a different position if the ``draggable`` parameter of the column is set to ``enable`` in the Config file.
```
Comment (MichalKinas) [inline digital-workspace/latest/config/index.md:73]:
```suggestion
    > * A column can be dragged to a different position if the ``draggable`` parameter of the column is set to ``true`` in the Config file.
```

Comment (MichalKinas) [reply digital-workspace/latest/config/index.md:73]: Other than that it looks good to me, I will approve whole PR when all comments are resolved

**Review by Prosune — APPROVED** [review]
Thank you for the review feedback
I am approving the PR based on the changes required for [DOCS-7551](https://alfresco.atlassian.net/browse/DOCS-7551). For suggestions beyond the scope of DOCS-7551, I have created a maintenance ticket [DOCS-7613](https://alfresco.atlassian.net/browse/DOCS-7613) and will be worked on later.

File: digital-workspace/4.3/develop/manage.md (hunk 1 above)
Comment (MichalKinas) [reply digital-workspace/4.3/develop/manage.md:27]: You can remove `ACA_BRANCH="2.10.0"` line, it doesn't have any replacement it's just obsolete

File: digital-workspace/4.3/develop/manage.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+    APP_CONFIG_OAUTH2_IMPLICIT_FLOW=true
+    APP_CONFIG_OAUTH2_SILENT_LOGIN=true
+    APP_CONFIG_OAUTH2_REDIRECT_SILENT_IFRAME_URI="{protocol}//{hostname}{:port}/assets/silent-refresh.html"
+    APP_CONFIG_OAUTH2_REDIRECT_LOGIN=/
+    APP_CONFIG_OAUTH2_REDIRECT_LOGOUT=/
+    APP_CONFIG_APPS_DEPLOYED="[{"name": "simpleapp"}]"
+    APP_CONFIG_LANDING_PAGE="/personal-files"
+
+    # CONTENT RELATED
+    ACA_BRANCH="develop"
```
[the remark this answers is not in the document]
Comment (MichalKinas) [reply digital-workspace/4.3/develop/manage.md:69]: Only the `ACA_BRANCH="develop"` part

File: digital-workspace/4.3/develop/manage.md (hunk 3 above)
Comment (MichalKinas) [reply digital-workspace/4.3/develop/manage.md:112]: `npm start content-ee` for plain ADW without APA

File: digital-workspace/4.3/develop/manage.md (hunk 4 above)
Comment (MichalKinas) [reply digital-workspace/4.3/develop/manage.md:120]: `npm run build content-ee` for ADW

File: digital-workspace/4.3/develop/manage.md [hunk 10]
```diff
@@ -0,0 +1,167 @@
...
+    Once the build has finished a new folder called `dist` is created inside the root directory of the project. Inside the `dist` directory there is a collection of files that represent the distribution of your application.
+
+## Remove modules before building
+
+To remove any modules from the distribution, access the `apps/content-ee/src/app/extensions.module.ts` file and remove the ones that are not needed.
+
+```java
+@NgModule({
+    imports: [
+        AosExtensionModule,
```
[the remark this answers is not in the document]
Comment (MichalKinas) [reply digital-workspace/4.3/develop/manage.md:134]: Currently the plugin presence depends on what is set in environment variables e.g. ` environment.plugins.aca_aos ? AosExtensionModule : []`

File: digital-workspace/4.3/develop/extensions.md [hunk 11]
```diff
@@ -0,0 +1,245 @@
...
+
+6. Edit the `apps/content-ee/src/app/extensions.module.ts` file to add the extension module to the application using the following format:
+
+    ```java
+    // Add the following import to the page.
+    import { MyExtensionModule } from '@alfresco-dbp/my-extension';
+    @NgModule({
+        imports: [
+            ...,
+            MyExtensionModule,
```
[the remark this answers is not in the document]
Comment (MichalKinas) [reply digital-workspace/4.3/develop/extensions.md:112]: That depends, if new extension should be also available and active this definition is fine, if client wants to enable it via environmental variables it should look like ` environment.plugins.customExtensions ? MyExtensionModule : []` and then `customExtensions` env variable has to also be added

**Pull request merged**